samples/app-anonymous-users/nodejs/README.md

```diff
@@ -0,0 +1,295 @@
+---
+page_type: sample
+description: This sample shows anonymous users support in meeting apps.
```

Comment:
Auth ?

Reply:
Are we updating existing sample for auth?

Reply:
This sample shows anonymous users support in `Teams` meeting apps.

Reply:
Fixed